internal/commands/push/command_test.go

```diff
@@ -417,79 +383,48 @@ diff2
-				expectedContents: "01:23:45 UTC INFO  Successfully pushed app changes\n",
-			},
-			{
-				description:      "That there is nothing to do when the diffs between app and draft do not exist",
-				outputs:          outputs{noDiffs: true},
-				expectedContents: "01:23:45 UTC INFO  Deployed app is identical to proposed version, nothing to do\n",
-			},
-			{
-				description:      "That no changes were pushed when the user rejects the diff",
-				outputs:          outputs{diffRejected: true},
-				expectedContents: "01:23:45 UTC INFO  No changes were pushed to your Realm application\n",
-			},
-			{
-				description:      "That no changes were pushed when the user chooses to keep its existing draft",
-				outputs:          outputs{diffRejected: true},
-				expectedContents: "01:23:45 UTC INFO  No changes were pushed to your Realm application\n",
-			},
-			{
-				description: "With a new app created and but in a dry run that the user should remove the dry run flag",
-				inputs:      inputs{DryRun: true},
-				outputs:     outputs{appCreated: true},
-				expectedContents: strings.Join(
-					[]string{
-						"01:23:45 UTC INFO  This is a new app. To create a new app, you must omit the 'dry-run' flag to proceed",
-						"01:23:45 UTC DEBUG Try running instead: realm-cli push\n",
-					},
-					"\n",
-				),
-			},
-			{
-				description:      "With a new app created and but in a dry run that the user should remove the dry run flag",
-				outputs:          outputs{appCreated: true},
-				expectedContents: "01:23:45 UTC INFO  This is a new app. You must create a new app to proceed\n",
-			},
-		} {
-			t.Run(tc.description, func(t *testing.T) {
-				out, ui := mock.NewUI()
-
-				cmd := &Command{inputs: tc.inputs, outputs: tc.outputs}
-
-				err := cmd.Feedback(nil, ui)
-				assert.Nil(t, err)
-
-				assert.Equal(t, tc.expectedContents, out.String())
-			})
-		}
-	})
-}
+// func TestPushFeedback(t *testing.T) {
+// 	t.Run("Feedback should print a message", func(t *testing.T) {
+// 		for _, tc := range []struct {
+// 			description      string
+// 			inputs           inputs
+// 			outputs          outputs
+// 			expectedContents string
+// 		}{
+// 			{
+// 				description:      "That changes were pushed successfully",
+// 				expectedContents: "01:23:45 UTC INFO  Successfully pushed app changes\n",
+// 			},
+// 			{
+// 				description:      "That there is nothing to do when the diffs between app and draft do not exist",
+// 				outputs:          outputs{noDiffs: true},
+// 				expectedContents: "01:23:45 UTC INFO  Deployed app is identical to proposed version, nothing to do\n",
+// 			},
+// 		} {
+// 			t.Run(tc.description, func(t *testing.T) {
+// 				out, ui := mock.NewUI()
+//
+// 				cmd := &Command{inputs: tc.inputs}
+//
+// 				err := cmd.Handler(nil, ui, cli.Clients{})
+// 				assert.Nil(t, err)
+//
+// 				assert.Equal(t, tc.expectedContents, out.String())
+// 			})
+// 		}
+// 	})
+// }
```

Comment:
[Q] Is this meant to still be here?

Reply:
Nope, thanks for catching this.  Tests for this are also now more flushed out in the the future PR for spinner support